cardano-node/src/Cardano/Node/Startup.hs

```diff
@@ -77,6 +78,17 @@ data StartupTrace blk =
 
   | StartupDBValidation
 
+  -- | Log that the block forging is being updated
+  | BlockForgingUpdate
```

Comment:
What does "the block forging is being updated" actually mean?

Reply:
Means that the Block forging credentials got updated, if you look at [here ](https://github.com/input-output-hk/cardano-node/pull/5383/commits/09804c79f6fe31c5b153aaa386555496d591deb5#diff-c0bfe28e2e76191ce40ccd02c48a10c280d94c6eaea663f5a61dc21ee6b6bc27R485) you'll see what it means